### DISCUSSION TITLE
Welcome to mdsplit Discussions!

### DISCUSSION THREAD

**Discussion by alandefreitas** [alandefreitas/mdsplit#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by lutasic** [comment]


**Reply by alandefreitas** [reply]
I'd be open to PRs but I'm not actively maintaining this project anymore. It doesn't match my philosophy for documentation anymore, so to speak.
